src/Http/Routing/test/UnitTests/EndpointRoutingMiddlewareTest.cs

```diff
@@ -58,9 +58,7 @@ public async Task Invoke_OnCall_WritesToConfiguredLogger()
         var expectedMessage = "Request matched endpoint 'Test endpoint'";
         bool eventFired = false;
 
-        var sink = new TestSink(
-            TestSink.EnableWithTypeName<EndpointRoutingMiddleware>,
-            TestSink.EnableWithTypeName<EndpointRoutingMiddleware>);
+        var sink = new TestSink(TestSink.EnableWithTypeName<EndpointRoutingMiddleware>);
```

Comment:
Is source.dot.net out of date? I can't see this constructor.

Reply:
... nevermind, I didn't see the default on the parameters. It might be good to use named parameters here.